src/main/redux/sagas/reader.ts

```diff
@@ -83,7 +74,13 @@ async function openReader(publication: Publication, manifestUrl: string) {
     const readerAppWindow = winRegistry.registerWindow(
         readerWindow,
         AppWindowType.Reader,
-    );
+        );
+
+    // If there are 2 win, record window position in the db
+    if (Object.keys(appWindows).length === 2) {
```

Comment:
I am wondering about the logic here ... shouldn't this be `Object.keys(appWindows).length >= 2`? (sorry if I am missing something obvious :)

Reply:
done

Reply:
What do you mean by "done"? The code hasn't changed :)

Reply:
it's logic

Reply:
You have to be more explicit, can you please explain the logic behind this code? Question: why not listen to `resize` and `move` events for reader windows when there are more than 2 windows in total (including the library window). Why restrict this to the first reader window only?

Reply:
call me on slack, I will show a demonstration on my computer with screen share

Reply:
Sorry, Slack screenshare only works with the Slack desktop app (I use their web app). Could you please articulate in a couple of sentences why we should only record the bounds of the first reader window, and not the subsequent ones?

Reply:
https://slack.com/intl/en-fr/downloads/mac, sorry but it's work for me

Reply:
i doesn't speak english very well on github , sorry

Reply:
You're doing fine, don't worry :)

Reply:
Resolving, because UX model / inconsistencies need to be resolved in separate PR (will file dedicated issue)